TODO.txt

```diff
@@ -8,44 +8,6 @@ Version 0.17
 * Finalize ``skimage.future.graph`` API.
 * Finalize ``skimage.future.manual_segmentation`` API.
 
-Version 0.18
-------------
-
-
-Version 0.19
-------------
-
-* Set ``start_label`` to 1 in ``slic`` and remove the DeprecationWarning.
-* In ``skimage/draw/draw.py`` remove ``circle'' and related tests.
-* In ``skimage/segmentation/morphsnakes.py`` remove ``circle_level_set'' and related tests.
-  * Also make sure to look in the function ``_init_level_set``
-* In ``skimage/morphology/_flood_fill.py`` replace the deprecated parameter
-  in flood_fill() ``inplace`` with ``in_place`` and update the tests.
```

Comment:
I didn't follow when or why this change was decided on... [pandas](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html) speaks 'inplace' so it seems more standard to me. Oh well.

Reply:
I don't remember, but I think it was for consistency with several other functions that use `in_place`. It looks like a number of functions under `future/graph` already have `in_place`, also `morphology.remove_small_holes` and `morphology.remove_small_objects.

Reply:
That said, most of those are still in future, so changes are probably expected there. I wonder if there are other prominent examples of `inplace` aside from pandas? 

I don't think SciPy or NumPy tend to use either, usually requiring and `out` or `output` argument to be specified instead.

Reply:
> I don't remember, but I think it was for consistency with several other functions that use `in_place`. It looks like a number of functions under `future/graph` already have `in_place`

Ok.

> also `morphology.remove_small_holes` and `morphology.remove_small_objects`

Right, but it looks like the argument is going away altogether for 1.0!
https://github.com/scikit-image/scikit-image/blob/7b0d37d4a3a3282a09b89175c0ee9197efb80b60/skimage/morphology/misc.py#L51
And going NumPy-style instead:
https://github.com/scikit-image/scikit-image/blob/7b0d37d4a3a3282a09b89175c0ee9197efb80b60/skimage/morphology/misc.py#L52